Source/WebKit/Shared/mac/ObjCObjectGraph.h

```diff
@@ -33,6 +33,19 @@ typedef struct objc_object* id;
 
 namespace WebKit {
 
+enum class ObjCType : uint8_t {
```

Comment:
There isn't really a good reason to move this to the header.  We could just migrate ObjCObjectGraph to use *.serialization.in then put this in a *.serialization.in with [Nested] because it's only used by the encoding of ObjCObjectGraph.  Also, ObjCObjectGraph basically just contains a variant of 8 things, so we could just completely remove it.